urgent-care/pom.xml

```diff
@@ -12,7 +12,7 @@
   <packaging>jar</packaging>
   <properties>
     <r4.version>1.0.18</r4.version>
-    <ee-artifacts.version>1.1.48</ee-artifacts.version>
+    <ee-apis-parent.version>3.0.0-SNAPSHOT</ee-apis-parent.version>
```

Comment:
Is this a SNAPSHOT because we need to test it with another PR?

Reply:
Yes. I'm waiting on Alec to review and Jenny to run fortify on API-1493.

Reply:
This has been updated to plain 3.0.1.